grain_size_tools/tools.py

```diff
@@ -31,7 +31,7 @@
 
 import os
 import numpy as np
-import plots as plots
+from . import plots
```

Comment:
Here, again, it is best practice to use relative, rather than global imports for methods and modules within the same package